Update grafana pd dashboard (#3343)
pingcap/docs-cn: grafana-pd-dashboard.md

以下为 PD Dashboard 监控说明：

- PD role：当前 PD 的角色
- Storage capacity：TiDB 集群总可用数据库空间大小
- Current storage size：TiDB 集群目前已用数据库空间大小
- Current storage usage：TiDB 集群存储空间的使用率
- Normal stores：处于正常状态的节点数目
- Number of Regions：当前集群的 Region 总量
- Abnormal stores：处于异常状态的节点数目，正常情况应当为 0

> - Region health：集群所有 Region 的状态，通常情况下，pending 或 down 的 peer 应该少于 100，miss 的 peer 不能一直大于 0， empty region 过多需及时打开 region merge

Reviewer: ```suggestion
- Region health：集群所有 Region 的状态。通常情况下，pending 或 down 的 peer 应该少于 100，miss 的 peer 不能一直大于 0，empty Region 过多需及时打开 Region Merge
```

- Current peer count：当前集群 peer 的总量
![PD Dashboard - Header](/media/pd-dashboard-header-v4.png)

## Cluster

- PD scheduler config：PD 调度配置列表

> - Cluster ID：集群的 cluster id， 唯一标识

Reviewer: ```suggestion
- Cluster ID：集群的 cluster id，唯一标识
```

- Current TSO：当前分配 TSO 的物理时间戳部分
- Current ID allocation：当前可分配 ID 的最大值
- Region label isolation level：不同 label 所在的 level 的 Region 数量
- Label distribution：集群中 TiKV 节点的 label 分布情况
![PD Dashboard - Cluster metrics](/media/pd-dashboard-cluster-v4.png)

## Operator

> - Schedule operator create：新创建的不同 operator 的数量， 单位 opm 代表一分钟内创建的个数 

Reviewer: ```suggestion
- Schedule operator create：新创建的不同 operator 的数量，单位 opm 代表一分钟内创建的个数 
```

- Schedule operator check：已检查的 operator 的次数，主要检查是否当前步骤已经执行完成，如果是，则执行下一个步骤
- Schedule operator finish：已完成调度的 operator 的数量
- Schedule operator timeout：已超时的 operator 的数量
- Schedule operator replaced or canceled：已取消或者被替换的 operator 的数量
- Schedule operators count by state：不同状态的 operator 的数量

> - Operator finish duration：已完成 operator 所花费的最长时间

Reviewer 2: ```suggestion
- Operator finish duration：已完成的 operator 所花费的最长时间
```

以下为 PD Dashboard 监控说明：

- PD role：当前 PD 的角色
- Storage capacity：TiDB 集群总可用数据库空间大小
- Current storage size：TiDB 集群目前已用数据库空间大小
- Current storage usage：TiDB 集群存储空间的使用率
- Normal stores：处于正常状态的节点数目
- Number of Regions：当前集群的 Region 总量
- Abnormal stores：处于异常状态的节点数目，正常情况应当为 0

> - Region health：集群所有 Region 的状态，通常情况下，pending 或 down 的 peer 应该少于 100，miss 的 peer 不能一直大于 0

Reviewer 2: 可以加一下“如果 empty region 堆积过多，会导致 balance region 调度失灵”

- Current peer count：当前集群 peer 的总量
![PD Dashboard - Header](/media/pd-dashboard-header-v4.png)

## Cluster

- PD scheduler config：PD 调度配置列表

> - Cluster ID: 集群的 cluster id, 唯一标识

Reviewer: 请将本文类似的所有英文冒号改为中文冒号。
```suggestion
- Cluster ID：集群的 cluster id, 唯一标识
```

- Current TSO: 当前分配 TSO 的物理时间戳部分
- Current ID allocation: 当前可分配 ID 的最大值
- Region label isolation level：不同 label 所在的 level 的 Region 数量
- Label distribution：集群中 TiKV 节点的 label 分布情况
![PD Dashboard - Cluster metrics](/media/pd-dashboard-cluster-v4.png)

## Operator

> - Schedule operator create：新创建的不同 operator 的数量, 单位 opm 代表一分钟内创建的个数 

Reviewer: ```suggestion
- Schedule operator create：新创建的不同 operator 的数量，单位 opm 代表一分钟内创建的个数 
```

- Schedule operator check：已检查的 operator 的次数，主要检查是否当前步骤已经执行完成，如果是，则执行下一个步骤
- Schedule operator finish：已完成调度的 operator 的数量
- Schedule operator timeout：已超时的 operator 的数量
- Schedule operator replaced or canceled：已取消或者被替换的 operator 的数量
- Schedule operators count by state：不同状态的 operator 的数量

> - 99% Operator finish duration：99% 已完成 operator 所花费的最长时间

Reviewer 3: 这个应该已经变成了 `Operator finish duration`？

- 50% Operator finish duration：50% 已完成 operator 所花费的最长时间

> - 99% Operator step duration：99% 已完成的 operator 的步骤所花费的最长时间

Reviewer 3: `Operator step duration`?

- Store leader count：每个 TiKV 实例上所有 leader 的数量
- Store Region count：每个 TiKV 实例上所有 Region 的数量

![PD Dashboard - Balance metrics](/media/pd-dashboard-balance-v4.png)

## Statistics - hot write

> - Hot Region's leader distribution：每个 TiKV 实例上写入热点的 leader 的数量

Reviewer 2: ```suggestion
- Hot Region's leader distribution：每个 TiKV 实例上成为写入热点的 leader 的数量
```

> - Total written bytes on hot leader Regions：每个 TiKV 实例上所有热点的 leader 的总的写入流量大小

Reviewer 2: ```suggestion
- Total written bytes on hot leader Regions：每个 TiKV 实例上所有成为写入热点的 leader 的总的写入流量大小
```

> - Hot write Region's peer distribution：每个 TiKV 实例上写入热点的 peer 的数量

Reviewer 2: ```suggestion
- Hot write Region's peer distribution：每个 TiKV 实例上成为写入热点的 peer 的数量
```

> - Total written bytes on hot peer Regions：每个 TiKV 实例上所有热点的 peer 的写入流量大小

Reviewer 2: ```suggestion
- Total written bytes on hot peer Regions：每个 TiKV 实例上所有成为写入热点的 peer 的写入流量大小
```

- Store Write rate bytes：每个 TiKV 实例总的写入的流量
- Store Write rate keys：每个 TiKV 实例总的写入 keys
- Hot cache write entry number：每个 TiKV 实例进入热点统计模块的 peer 的数量
- Selector events： 热点调度中选择器的事件发生次数

> - Direction of hotspot move leader：热点调度 leader 的调度方向，正数代表调入，负数代表调出

Reviewer 2: ```suggestion
- Direction of hotspot move leader：热点调度中 leader 的调度方向，正数代表调入，负数代表调出
```

> - Direction of hotspot move peer： 热点调度 peer 的调度方向，正数代表调入，负数代表调出

Reviewer 2: ```suggestion
- Direction of hotspot move peer： 热点调度中 peer 的调度方向，正数代表调入，负数代表调出
```

> - Direction of hotspot move peer： 热点调度中 peer 的调度方向，正数代表调入，负数代表调出

Reviewer: ```suggestion
- Direction of hotspot move peer：热点调度中 peer 的调度方向，正数代表调入，负数代表调出
```

![PD Dashboard - Hot write metrics](/media/pd-dashboard-hotwrite-v4.png)

## Statistics - hot read

> - Hot Region's leader distribution：每个 TiKV 实例上读取热点的 leader 的数量

Reviewer 2: ```suggestion
- Hot Region's leader distribution：每个 TiKV 实例上成为读取热点的 leader 的数量
```

> - Total read bytes on hot leader Regions：每个 TiKV 实例上所有热点的 leader 的总的读取流量大小

Reviewer 2: ```suggestion
- Total read bytes on hot leader Regions：每个 TiKV 实例上所有成为读取热点的 leader 的总的读取流量大小
```

- Store leader count：每个 TiKV 实例上所有 leader 的数量
- Store Region count：每个 TiKV 实例上所有 Region 的数量

![PD Dashboard - Balance metrics](/media/pd-dashboard-balance-v4.png)

## Statistics - hot write

> - Hot Region's leader distribution：每个 TiKV 实例上是写入热点的 leader 的数量

Reviewer 2: ```suggestion
- Hot Region's leader distribution：每个 TiKV 实例上写入热点的 leader 的数量
```

- Total written bytes on hot leader Regions：每个 TiKV 实例上所有热点的 leader 的总的写入流量大小

> - Hot write Region's peer distribution：每个 TiKV 实例上是写入热点的 peer 的数量

Reviewer 2: ```suggestion
- Hot write Region's peer distribution：每个 TiKV 实例上写入热点的 peer 的数量
```

- Total written bytes on hot peer Regions：每个 TiKV 实例上所有热点的 peer 的写入流量大小
- Store Write rate bytes: 每个 TiKV 实例总的写入的流量
- Store Write rate keys: 每个 TiKV 实例总的写入 keys
- Hot cache write entry number: 每个 TiKV 实例进入热点统计模块的 peer 的数量
- Selector events: 热点调度中选择器的事件发生次数
- Direction of hotspot move leader: 热点调度 leader 的调度方向，正数代表调入，负数代表调出

> - Direction of hotspot move peer:  热点调度 peer 的调度方向，正数代表调入，负数代表调出

Reviewer: ```suggestion
- Direction of hotspot move peer：热点调度 peer 的调度方向，正数代表调入，负数代表调出
```

- Balance Region scheduler：balance-region scheduler 的状态

> - Namespace checker：namespace checker 的状态

Reviewer 2: 我们还需要保留 namespace checker 么

- Balance Region scheduler：balance-region scheduler 的状态
- Namespace checker：namespace checker 的状态
- Replica checker：replica checker 的状态

> - Rule checker：replica checker 的状态

Reviewer 4: 104 和 105 行写的都是 “replica checker 的状态”，笔误？

## TiDB

> - PD Server TSO handle time and Client recv time: 从 PD 开始处理 TSO 请求到 client 端接收到 TSO 的总耗时

Reviewer 2: ```suggestion
- The duration between PD Server handle TSO and Client recv : 从 PD 开始处理 TSO 请求到 client 端接收到 TSO 的总耗时
```

Author: 这个是监控面板的名字。

- Handle requests count：TiDB 的请求数量
- Handle requests duration：每个请求所花费的时间，99% 的情况下，应该小于 100ms

![PD Dashboard - TiDB metrics](/media/pd-dashboard-tidb-v4.png)

## Heartbeat

> - Heartbeat region evnent QPS: 心跳处理 region 的 QPS, 包括更新缓存和持久化

Reviewer 2: ```suggestion
- Heartbeat region event QPS: 心跳处理 region 的 QPS, 包括更新缓存和持久化
```
